Marlin/src/module/probe.cpp

```diff
@@ -438,6 +438,8 @@ bool set_probe_deployed(const bool deploy) {
         KEEPALIVE_STATE(PAUSED_FOR_USER);
         wait_for_user = true;
         while (wait_for_user) idle();
+        lcd_reset_alert_level();
+        lcd_quick_feedback(true);
```

Comment:
If these two calls are needed, then it’s a bug in the LCD code, and should be fixed elsewhere, such as in `lcd_reset_status`. The quick feedback in particular should already have been called in response to the click that cleared `wait_for_user`.  The reset of the alert level is questionable. It should perhaps be reset by `lcd_reset_status` but bears closer examination to understand how the alert level is employed and where it is most appropriate to clear it. High-level alerts should not be overridden by other messages.

Reply:
I think probably the correct answer is to change `lcd_setalertstatusPGM` to `lcd_setstatusPGM` just above here.